### PR TITLE
Updated Training related structures and methods

### PR DESCRIPTION
**Pull request by manspro**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -16,48 +16,48 @@ const (
...
-	// вставьте ваш код ниже
-	...
+	return float64(t.Action) * LenStep / MInKm
 }
 
 // meanSpeed возвращает среднюю скорость бега или ходьбы.
 func (t Training) meanSpeed() float64 {
-	// вставьте ваш код ниже
-	...	
+	return t.distance() / (t.Duration.Minutes() / MinInHours)
```
[the remark this answers is not in the document]
Comment (manspro) [reply main.go:35]: done

File: main.go [hunk 2]
```diff
@@ -145,46 +139,35 @@ const (
...
 }
 
 // meanSpeed возвращает среднюю скорость при плавании.
-// Формула расчета:
-// длина_бассейна * количество_пересечений / м_в_км / продолжительность_тренировки
 // Это переопределенный метод Calories() из Training.
 func (s Swimming) meanSpeed() float64 {
-	// вставьте ваш код ниже
-	...
+	return float64(s.LengthPool*s.CountPool) / MInKm / s.Duration.Hours()
```
[the remark this answers is not in the document]
Comment (manspro) [reply main.go:150]: done

File: main.go [hunk 3]
```diff
@@ -145,46 +139,35 @@ const (
...
-	...
+	return (s.meanSpeed() + SwimmingCaloriesMeanSpeedShift) * SwimmingCaloriesWeightMultiplier * s.Weight * s.Duration.Hours()
 }
 
 // TrainingInfo returns info about swimming training.
 // Это переопределенный метод TrainingInfo() из Training.
 func (s Swimming) TrainingInfo() InfoMessage {
-	// вставьте ваш код ниже
-	...
+	return s.Training.TrainingInfo()
```
[the remark this answers is not in the document]
Comment (manspro) [reply main.go:162]: done

File: main.go [hunk 4]
```diff
@@ -125,15 +121,13 @@ type Walking struct {
 // * 0.029 * вес_спортсмена_в_кг) * время_тренировки_в_часах * мин_в_ч)
 // Это переопределенный метод Calories() из Training.
 func (w Walking) Calories() float64 {
-	// вставьте ваш код ниже
-	...
+	return (CaloriesWeightMultiplier*w.Weight + (math.Pow(w.meanSpeed()*KmHInMsec, 2)/(w.Height/CmInM))*CaloriesSpeedHeightMultiplier*w.Weight) * w.Duration.Minutes()
```
[the remark this answers is not in the document]
Comment (manspro) [reply main.go:124]: done

**Review by just-musha — APPROVED** [review]
Спасибо, супер! Принято

**Pull request merged**